### PR TITLE
Updated CS and SK login messages

### LINKED ISSUE #27527: CS and SK localized messages need an update
### Description

Will be posting a PR with the changes shortly.

### Discussion

_No response_

### Motivation

#26943 and #27151 changed login messages.

### Details

_No response_

### PR DESCRIPTION
**Pull request by Lamak121**
Closes #27527

_Passkey_ translation was taken from [this](https://support.apple.com/en-us/102195) Apple article that is written in all the three relevant languages.

### CODE REVIEW

File: themes/src/main/resources-community/theme/base/login/messages/messages_sk.properties [hunk 1]
```diff
@@ -472,25 +472,25 @@ recovery-codes-download-file-date=Tieto kódy boli vygenerované dňa
...
-webauthn-passwordless-display-name=Bezpečnostný kľúč
-webauthn-passwordless-help-text=Použite svoj bezpečnostný kľúč na prihlásenie bez hesla.
-webauthn-login-title=Prihlásenie bezpečnostným kľúčom
-webauthn-registration-title=Registrácia bezpečnostným kľúčom
-webauthn-available-authenticators=Dostupné bezpečnostné kľúče
+webauthn-display-name=Prihlasovací kľúč
+webauthn-help-text=Použite svoj prihlasovací kľúč na prihlásenie.
+webauthn-passwordless-display-name=Prihlasovací kľúč
+webauthn-passwordless-help-text=Použite svoj prihlasovací kľúč na prihlásenie bez hesla.
+webauthn-login-title=Prihlásenie prihlasovacom kľúčom
```
Comment (tondrusk) [inline themes/src/main/resources-community/theme/base/login/messages/messages_sk.properties:479]:
```suggestion
webauthn-login-title=Prihlásenie prihlasovacím kľúčom
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_sk.properties [hunk 2]
```diff
@@ -472,25 +472,25 @@ recovery-codes-download-file-date=Tieto kódy boli vygenerované dňa
...
+webauthn-registration-title=Registrácia prihlasovacom kľúčom
```
Comment (tondrusk) [inline themes/src/main/resources-community/theme/base/login/messages/messages_sk.properties:480]:
```suggestion
webauthn-registration-title=Registrácia prihlasovacím kľúčom
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_sk.properties [hunk 3]
```diff
@@ -472,25 +472,25 @@ recovery-codes-download-file-date=Tieto kódy boli vygenerované dňa
...
-webauthn-error-registration=Nepodarilo sa zaregistrovať bezpečnostný kľúč.<br/> {0}
-webauthn-error-api-get=Nepodarilo sa overiť pomocou bezpečnostného kľúča.<br/> {0}
-webauthn-error-different-user=Prvý overený používateľ nie je ten, ktorý bol overený bezpečnostným kľúčom.
-webauthn-error-auth-verification=Výsledok overenia bezpečnostného kľúča je neplatný.<br/> {0}
-webauthn-error-register-verification=Výsledok registrácie bezpečnostného kľúča je neplatný.<br/> {0}
-webauthn-error-user-not-found=Neznámy používateľ overený bezpečnostným kľúčom.
+webauthn-error-title=Chybný prihlasovací kľúč
+webauthn-error-registration=Nepodarilo sa zaregistrovať prihlasovací kľúč.<br/> {0}
+webauthn-error-api-get=Nepodarilo sa overiť pomocou prihlasovacieho kľúča.<br/> {0}
+webauthn-error-different-user=Prvý overený používateľ nie je ten, ktorý bol overený prihlasovacom kľúčom.
```
Comment (tondrusk) [inline themes/src/main/resources-community/theme/base/login/messages/messages_sk.properties:490]:
```suggestion
webauthn-error-different-user=Prvý overený používateľ nie je ten, ktorý bol overený prihlasovacím kľúčom.
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_sk.properties [hunk 4]
```diff
@@ -472,25 +472,25 @@ recovery-codes-download-file-date=Tieto kódy boli vygenerované dňa
...
+webauthn-error-auth-verification=Výsledok overenia prihlasovacieho kľúča je neplatný.<br/> {0}
+webauthn-error-register-verification=Výsledok registrácie prihlasovacieho kľúča je neplatný.<br/> {0}
+webauthn-error-user-not-found=Neznámy používateľ overený prihlasovacom kľúčom.
```
Comment (tondrusk) [inline themes/src/main/resources-community/theme/base/login/messages/messages_sk.properties:493]:
```suggestion
webauthn-error-user-not-found=Neznámy používateľ overený prihlasovacím kľúčom.
```

**Review by tondrusk — COMMENTED** [review]
I found some typos and I wanted to ask why there is such difference between these two very similar languages (or words, let's say)?

The Czech word "přístupový" can be translated to Slovak to "prístupový" and on the other hand, the Slovak word "prihlasovací" have a Czech equivalent of "přihlasovací".

přístupový/prístupový could be described as **access**.
přihlasovací/prihlasovací could be described as **log in**/**sign in**.

Depending on what should be the word described as, we should refer to access or log in/ sign in.

I looked up what word is being translated and it's **Passkey**.

**Comment by Lamak121** [comment]
> I found some typos and I wanted to ask why there is such difference between these two very similar languages (or words, let's say)?
> 
> The Czech word "přístupový" can be translated to Slovak to "prístupový" and on the other hand, the Slovak word "prihlasovací" have a Czech equivalent of "přihlasovací".
> 
> přístupový/prístupový could be described as **access**. přihlasovací/prihlasovací could be described as **log in**/**sign in**.
> 
> Depending on what should be the word described as, we should refer to access or log in/ sign in.
> 
> I looked up what word is being translated and it's **Passkey**.

Sorry for the typo. I will defend myself with the fact that the native speaker (which I am not, I am Czech) that I consulted for this declined the adjective incorrectly as well. I confused the locative with the instrumental and your declension is correct.

As for the translation of "passkey", the reason for the difference is the [Apple article](https://support.apple.com/en-us/102195) I cited in the PR description. If you look at the Czech and Slovak version respectively, the translations differ. I kept it that way.

**Comment by Lamak121** [comment]
> I found some typos and I wanted to ask why there is such difference between these two very similar languages (or words, let's say)?
> 
> The Czech word "přístupový" can be translated to Slovak to "prístupový" and on the other hand, the Slovak word "prihlasovací" have a Czech equivalent of "přihlasovací".
> 
> přístupový/prístupový could be described as **access**. přihlasovací/prihlasovací could be described as **log in**/**sign in**.
> 
> Depending on what should be the word described as, we should refer to access or log in/ sign in.
> 
> I looked up what word is being translated and it's **Passkey**.

Hi, I have to disagree with you on the typos. The correct declension of the word "prihlasovací" here is "prihlasovacím" since the connected noun "kľúčom" is of the instrumental form and the adjective follows. See [this](https://www.eduself.sk/slovensky-jazyk/tvary-slova/prihlasovac%C3%AD) article that declines the word in all the possible ways.

As for the translation of "passkey", in the PR I cited [this](https://support.apple.com/en-us/102195) Apple article where if you look at the Czech and Slovak versions respectively, the translations of the word differ. I kept it that way.

**Comment by tondrusk** [comment]
> Sorry about the typos. You are right the noun is of the instrumental form and the adjective follows. [This](https://www.eduself.sk/slovensky-jazyk/tvary-slova/prihlasovac%C3%AD) article agrees with you. Just to make sure, I also consulted a native speaker about this.
> 
> As for the translation of "passkey", in the PR I cited [this](https://support.apple.com/en-us/102195) Apple article where if you look at the Czech and Slovak versions respectively, the translations of the word differ. I kept it that way.

If we want to rely on some "source of truth" which could be the Apple article, I personally don't mind using a bit different translations.

Just to clarify (and I shoud've done it in the first place), I'm a native Slovak speaker currently living in Czech Republic, so I hope my language knowledge is naturally correct.

The suggestions should still be applied.

**Comment by Lamak121** [comment]
> > Sorry about the typos. You are right the noun is of the instrumental form and the adjective follows. [This](https://www.eduself.sk/slovensky-jazyk/tvary-slova/prihlasovac%C3%AD) article agrees with you. Just to make sure, I also consulted a native speaker about this.
> > As for the translation of "passkey", in the PR I cited [this](https://support.apple.com/en-us/102195) Apple article where if you look at the Czech and Slovak versions respectively, the translations of the word differ. I kept it that way.
> 
> If we want to rely on some "source of truth" which could be the Apple article, I personally don't mind using a bit different translations.
> 
> Just to clarify (and I shoud've done it in the first place), I'm a native Slovak speaker currently living in Czech Republic, so I hope my language knowledge is naturally correct.
> 
> The suggestions should still be applied.

Great that we agree! Should I apply all the suggestions individually or is there a way to do it as one commit? I'm pretty new to GitHub, sorry.

**Comment by jonkoops** [comment]
@Lamak121 you should be able to batch the changes by going to the 'Files changed' panel, see the [GitHub documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request) for more details.

**Review by tondrusk — APPROVED** [review]

**Review by hmlnarik — APPROVED** [review]
Approving, thanks for the contribution!

**Pull request merged**